src/main/resources/db/changelog/changes/v1.0.0/sql/2021-07-22-01__alter-itm-contrib-opt-conf-table.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE item_contribution_options_configuration ADD COLUMN id UUID;
+ALTER TABLE item_contribution_options_configuration DROP CONSTRAINT item_contribution_options_configuration_pkey CASCADE;
+ALTER TABLE item_contribution_options_configuration ADD CONSTRAINT pk_item_contribution_options_configuration PRIMARY KEY (id);
```

Comment:
Imaging there are records in `item_contribution_options_configuration` table. What will be in `id` column for those record?

Try to reproduce the case locally and see what happends

Reply:
@dmtkachenko @maryna-steshenko Wouldn't it be suitable just to map existing PK `central_server_id` to `id` in DTO? I believe this is the simplest way to introduce `id` property in response without changing too much